pkg/devfile/parser/data/common/types.go

```diff
@@ -181,36 +181,37 @@ type Composite struct {
 	Parallel bool `json:"parallel,omitempty"`
 }
 
-// Git Project's Git source
-type Git struct {
+type CheckoutFrom struct {
```

Comment:
the 2.0.0/types.go also needs to  be updated

Reply:
deleted most from 2.0.0/types.go it looks like they are not used anywhere